mscore/scoreview.cpp

```diff
@@ -320,6 +320,15 @@ void ScoreView::objectPopup(const QPoint& pos, Element* obj)
       popup->addAction("Debugger")->setData("list");
 #endif
 
+#if 0
```

Comment:
If this code is not enabled anyway, would it perhaps be better to add it in a separate commit?

Reply:
You mean just put a TODO comment here but leave out the code?  I like to add a rough outline of the expected code where possible when I add a TODO, and it seems Werner favored that approach too.  Or maybe you mean, just add the TODO and disabled code as a separate commit within this same PR?

However, in this case I am fine removing it, because it actually appears the problem is not ours, it is a Qt bug.  As per one of the comments in the issue, the failure of the screenreader to read the items on the popup menu that are duplicated in the main menu is because of https://bugreports.qt.io/browse/QTBUG-79858.  So I will just remove this section, which would have just been a workaround even if I had been able to get it to work.

Reply:
Let's better remove this section then, the other part of this PR is valuable itself, and the information added here is anyway present in the issue trackers, both our and Qt's ones.

Reply:
This disabled code seems to have returned with the latest push to this branch.